lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,48 @@
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTOGONALITIES = %i[length_x length_y].freeze
+        CUT_DIRECTION_ERROR = 'Not valid cut direction, should one of thes: :length_x :length_y'.freeze
+
+        def initialize(column:, footing:, effective_height:, load_from_column:, cut_direction:)
+          @column = column
+          @footing = footing
+          @load_from_column = load_from_column
+          @effective_height = effective_height.to_f
+
+          raise ArgumentError, CUT_DIRECTION_ERROR unless ORTOGONALITIES.any?(cut_direction)
+
+          @cut_direction = cut_direction
```

Comment:
I don't think this is the right translation for what we need. Maybe `section_direction`?

Reply:
agree